daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -1709,6 +1709,41 @@ class RazerBasiliskV3Pro35KWireless(RazerBasiliskV3Pro35KWired):
     USB_PID = 0x00CD
 
 
+class RazerBasiliskV3_35K(__RazerDevice):
+    """
+    Class for the Razer Basilisk V3 35K
+    """
+
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Basilisk_V3_35K_000000000000-if0(1|2)-event-kbd')
```

Comment:
I think this should be `Razer_Basilisk_V3_35K-if0(1|2)-event-kbd`  - can you check `ls -lah /dev/input/by-id/` with your device?